packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -629,6 +630,11 @@ const NewXostorForm = decorate([
           replication,
         } = this.state
 
+        await confirm({
+          title: _('xostorCreation'),
+          body: _('restartRequiredWarningMessage'),
+        }).then(() => createXostorSr())
+
```

Comment:
```suggestion
        await confirm({
          title: _('xostorCreation'),
          body: _('restartRequiredWarningMessage'),
        })
```

Reply:
AFAIK, `confirm` will reject if the action is canceled, will that cause issues? (e.g. like the action button showing an error)

Reply:
I believe tgat `ActionButton` already correctly handles the error thrown by the `confirm` modal.

Reply:
After testing, no error appears when canceling the creation